### ISSUE TITLE
clustering.py missing

### ISSUE THREAD

**Issue by THANNAGA** [skumar9876/Hierarchical-DQN#1]
Thank you for this code. 
Is the project still active, and does the code indeed reproduce the result of the paper?
It looks like the clustering.py script is missing.

**Comment by GoingMyWay** [comment]
Indeed, `clustering.py` is missing

**Comment by skumar9876** [comment]
I apologize for not seeing this earlier! Yes, clustering.py is missing -
the Hierarchical DQN agent was used with subgoals formed by clustering the
state space for an exploratory project. The purpose of this repo is to
provide an implementation of a Hierarchical DQN agent, however this agent
has not been tested with the experiments done in Kulkarni et al. The
train_dqn file should be rewritten to remove the use of the clustering
module.

**Comment by YantianZha** [comment]
@skumar9876 I agree with you. The original H-DQN uses the same state spaces for both meta-controller and low-level controller. So there is no need to use clustering. 

I'm trying to use your code. So have you rewritten the code without the clustering module (i.e., the purely reimplementation of H-DQN)?

Thanks!

**Comment by momofive** [comment]
Excuse me, I wonder have you rewritten the code without the clustering module? Thanks!
